udata/core/user/models.py

```diff
@@ -44,6 +44,7 @@ class User(WithMetrics, UserMixin, db.Document):
     email = db.StringField(max_length=255, required=True, unique=True)
     password = db.StringField()
     active = db.BooleanField()
+    fs_uniquifier = db.StringField(max_length=64, unique=True)
```

Comment:
New mandatory user's attribute. The `fs_uniquifier` is now used in most auth mechanisms.
For instance, when the user change its password, it is used to invalidate existing sessions.
The `fs_uniquifier` is used instead of the `id` in permissions. This leads to the several following changes in our permissions.